app/components/Notifier/Notifier.jsx

```diff
@@ -73,13 +75,24 @@ export default class Notifier extends Component
                 updateNotification({ id, response: 'deny' })
             };
         }
+         logger.info('==================-=-------------------=============-------');
+         logger.info(authReqEl);
+         logger.info(JSON.stringify(authReqEl));
 
 
         return (
             <Row hasMinHeight className={ styles.container }>
                 <MessageBox messageType={type}>
                     <Row verticalAlign="middle" align="center">
-                        <Text className={CLASSES.NOTIFIER_TEXT}>{ text }</Text>
+                        <Column align="left">
+                          {
+                            authReqEl && authReqEl
```

Comment:
This line throws an error that component `Column` is expecting `ReactNode` but instead was passed `children`, however, `authReqEl` is an React element object:
```
{ _owner: null,
  key: null,
  props: { children: [ [Object], [Object], [Object], [Object], [Object] ] },
  ref: null,
  type: 'div' }
```

Reply:
Yo, so here it's needed to rebuild the react components with react.createElement. There's some looping over this that needs done eg (a rough version):

```jsx
 const reReact = ( thing ) =>
        {
            const reacter = {
            };
            Object.keys( thing ).forEach( key => {
                if( key === 'type' )
                {
                    return reacter[key] = thing[key];
                }

                if( key === 'props' )
                {
                    Object.keys( thing[key] ).forEach( prop => {

                        if( prop === 'children' )
                        {
                            return reacter.children = thing.props.children;
                        }

                        reacter.props = {
                            [prop] : thing[key][prop]
                        }
                     } )

                }
            } )
            const newChildArray = [];
            let el;
            if( Array.isArray( reacter.children ) )
            {
                reacter.children.forEach( child => newChildArray.push( reReact(child) ))
                reacter.children = newChildArray;
            }

            console.log( Array.isArray( reacter.children ) , 'arrrrrrrr??????????')
            return React.createElement( reacter.type, reacter.props, reacter.children);
        }
        
        const restruct = reReact(authReqEl);
        
        console.log('and so we have:', restruct )
```

the other thing here is that instead of adding `authReq` stuff to `Notifier`, as that's SAFE specific. This should come from / be detected from the props. eg: Either there's a text prop, OR, if a 'textNode' is passed, it parses the react node if needed. (basically as you have, but a more generic naming)